amr-wind/ocean_waves/OceanWaves.H

```diff
@@ -0,0 +1,78 @@
+#ifndef OCEAN_WAVES_H
+#define OCEAN_WAVES_H
+
+#include "amr-wind/core/Physics.H"
+#include "amr-wind/core/FieldRepo.H"
+
+/** \defgroup immersed boundary Immersed boundary module
+ *  Immersed boundary class
+ *  \ingroup immersed_boundary
+ */
```

Comment:
are these comments meant for the future? If not, we should remove the referencees to IB